packages/react-scripts/scripts/eject.js

```diff
@@ -302,6 +302,16 @@ inquirer
     console.log(green('Ejected successfully!'));
     console.log();
 
+    console.log(cyan('Adding ejected files to git staging area...'));
```

Comment:
I think this message should be something like `Adding ejected files to git...` or `Staging ejected files for commit...` or something along those lines. I'm not 100% sure how best to phrase that though.

Reply:
Sure, that works for me.